src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.component.html

```diff
@@ -0,0 +1,6 @@
+<p-splitButton 
+    flat 
+    class="edit-page-toolbar__actions"
```

Comment:
Base of what I said before, you don't need it.

Reply:
I'll leave this so I don't have to change the tests, anyway I don't think this would affect anything